lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -130,57 +135,58 @@ void Interpreter::fwdConvolutionInst_I8Impl(Value *inV, Value *outV,
   // multiplication part of the calculation.
   float matMulScale = inScale * filterScale;
 
-  size_t inChannels = idim.c;
-
   // For each input in the batch:
   for (size_t n = 0; n < idim.n; n++) {
+    // For each group of input channels:
+    for (size_t g = 0; g < group; g++) {
+      // For each layer in the output tensor:
```

Comment:
from the fp version:
 // For each output channel in the group:
      for (size_t d = g * outCperG; d < (g + 1) * outCperG; d++) {